Library/Homebrew/livecheck/strategy/xml.rb

```diff
@@ -72,6 +72,30 @@ def self.parse_xml(content)
           end
         end
 
+        # Retrieves the stripped inner text of an `REXML` element. Returns
+        # `nil` if the optional child element doesn't exist or the text is
+        # blank.
+        # @param element [REXML::Element] an `REXML` element to retrieve text
+        #   from, either directly or from a child element
+        # @param child_path [String, nil] the XPath of a child element to
+        #   retrieve text from
+        # @return [String, nil]
+        sig {
+          params(
+            element:    REXML::Element,
+            child_path: T.nilable(String),
+          ).returns(T.nilable(String))
+        }
+        def self.element_text(element, child_path = nil)
+          element = element.get_elements(child_path).first if child_path.present?
+          return if element.nil?
```

Comment:
`return if child_path.blank?` as first line seems nicer.

Also, longer-term, it's even nicer still to learn more on Sorbet and not make `child_path` nil able.

Reply:
Using `return if child_path.blank?` would effectively make `child_path` a required parameter. As mentioned in the PR description, I intentionally made `child_path` an optional parameter, so `#element_text` can also be used to retrieve text from the provided element instead. This allows us to use `Xml.element_text(item)` to avoid `item.text.strip.presence`, if/when needed in the future.

Reply:
Ok, I see, I misread before: the `element` variable is overwritten if `child_path` is passed.